blog/admin.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib import admin
+from django.contrib.auth.models import Group
+from django.contrib.auth.admin import UserAdmin
+from .models import User, Post, Commentary
+
+admin.site.unregister(Group)
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):
+    search_fields = ("username",)
+
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        (
+            (
+                "Additional info",
+                {
+                    "fields": (
+                        "first_name",
+                        "last_name",
+                        "email",
+                    )
+                },
+            ),
+        )
+    )
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = ["title", "created_time", ]
+    search_fields = ("owner",)
+    list_filter = ("created_time",)
+
+
+admin.site.register(Commentary)
```

Comment:
Use either a function or decorator-based approach while registering your models in the `admin.py`